src/runtime/components/forms/Form.vue

```diff
@@ -1,18 +1,18 @@
 <template>
-  <form @submit.prevent="onSubmit">
+  <form :id="formId" @submit.prevent="onSubmit">
```

Comment:
Why add an `id` to the form here?

Reply:
I removed it, it was a leftover from a test to check for hydration issues.